README.md

```diff
@@ -159,6 +161,19 @@ Of type `u64`, initialized to `0`.
 
 Expresses the portion of Surplus Weight which has been refunded. Not used on XCM platforms which do not require payment for execution.
 
+### **3.10** Transact Status
+
+Of type `MaybeErrorCode`, initialized to `MaybeErrorCode::Success`.
+
+Expresses the status when checking the versions between the origin, destination chain, and the requested pallet located on the destination chain.
```

Comment:
This is not what the transact status register does. It reports the dispatch result of a `Transact` instruction after dispatching the encoded call.